app/assets/stylesheets/metrics/_interval-filter.scss

```diff
@@ -0,0 +1,12 @@
+.period-input {
+  @include flex-row-start;
+}
+
+.period-selection{
+  label.numeric{
+    width: 13rem;
+  }
+  input.numeric{
+    margin: 0 0.2rem;
+  }
+}
```

Comment:
Is this fully responsive? The 13 rem looks weird.

Reply:
yep, it is just a with to separate the input from the label